wsgi/odyssy/tag/views.py

```diff
@@ -0,0 +1,54 @@
+from announcement.models import Announcement
+from news.models import News
+from basic.models import PhotoExtended
+
+from django.shortcuts import render
+from django.core.paginator import Paginator, EmptyPage
+
+
+def simple_tag(request, tag_name):
+    """ View for simple tag listing """
+    latest_announcement = Announcement.get_latest_announcement_tag(tag_name, 5)
+    latest_news = News.get_latest_news_tag(tag_name, 5)
+    latest_images = PhotoExtended.get_latest_images_tag(tag_name, 5)
+    args = {
+        'latest_announcement': latest_announcement,
+        'latest_news': latest_news,
+        'latest_images': latest_images,
+    }
+    return render(request, 'tag/simple_tag.html', args)
+
+
+def paginate_view(request, query_set, num_items):
```

Comment:
This method can be used by other views too. Put it in `basic.utils` and make all the views use it.